### ISSUE TITLE
Everything is darker compared to JMP

### ISSUE THREAD

**Issue by cjghost2000** [prayag17/JellyPlayer#291]
Is there any way to tweak / fix this ? 
Examples below also look a lot brighter than what I'm seeing, the Drive ones are basically black when viewing on JP
hdr is off. checked nvidia colour settings. no icc profile
JMP Left - JP Right
![image](https://github.com/prayag17/JellyPlayer/assets/25965419/c6130b38-47e0-4c01-a041-f37c4b238fc9)
![image](https://github.com/prayag17/JellyPlayer/assets/25965419/93f36288-e0b2-4471-8c9e-cc2d57303291)
![image](https://github.com/prayag17/JellyPlayer/assets/25965419/4d24570b-aed7-4779-94d1-af09ba09d248)
![jellyplayer](https://github.com/prayag17/JellyPlayer/assets/25965419/0ec9d96e-4f83-487c-9d3c-2df32d58c525)

**Comment by prayag17** [comment]
Are you using the appimage?

**Comment by cjghost2000** [comment]
> Are you using the appimage?

I used the EXE "JellyPlayer_0.0.5-dev_x64-setup.exe" from releases I'm also on Windows

**Comment by prayag17** [comment]
You should look into your window's playback settings

**Comment by cjghost2000** [comment]
Yeah, I've gone through Video Playback, Turned HDR on and off again, reset nvidia control panel colour settings, reinstalled drivers to make sure settings were reset, reset colour profiles under Colour Management, Recalibrated display in windows, made sure Colour Filters were off in windows i just cant seem to get it to look right sadly, i also found that audio tracks don't switch and i cant play anything with the AC3 Audio Codec

**Comment by prayag17** [comment]
have you turned off 
![image](https://github.com/prayag17/JellyPlayer/assets/55829513/04a186a6-4181-4e70-97c3-12587be01346)

**Comment by cjghost2000** [comment]
yeah that setting is off, I even toggled it on and off and it didn't seem to change anything

**Comment by prayag17** [comment]
Thats weird, I cant seem to replicate this "bug" its most likely an issue with your machine. But I am gonna keep this issue open for now.

**Comment by cjghost2000** [comment]
> Also just to confirm its a bug with JellyPlayer you can play the same files in the updated edge browser

weirdly enough just tried it in edge like you said and its dark there too until i turn off hardware acceleration then it looks normal, firefox and brave are also fine with and without hardware accel

**Comment by prayag17** [comment]
Yeah that's just windows being windows. Edge also uses webview2 same as JellyPlayer and hence the issue arises there too.

**Comment by qvattr0** [comment]
I actually was going to report having the same issues as the OP (if I recall correctly, I've brought this issue up initially a long time ago as well). Video enhancements are turned off and all the media foundation codecs are installed as well. Still getting a darkened image compared to JMP.

**Comment by prayag17** [comment]
This is not really a JellyPlayer issue but a bug with WebView2 on windows, you can test this out by playing the same files in the new edge browser. One potential work around for now is to turn off hardware acceleration for video playback

**Comment by qvattr0** [comment]
After extensive troubleshooting, I was able to figure the reason for why this occurs (well, I've discovered a solution). Unfortunately, this solution only applies to WebView2 playback on Edge, and I was not able to figure out a way to change the necessary parameters in JellyPlayer. 

I've started with exploring Jellyfin playback on Edge. As expected, the image was darker there despite not using Jellyplayer; therefore, the issue is indeed with WebView2. Now, the immediate solution for this issue is turning off hardware acceleration (graphics acceleration in Edge). This is not an ideal solution because I wasn't able to Direct Play any of my HEVC videos (interestingly, the playback of H264 files did not exhibit the wrong gamma). An alternative solution I've come across was changing the ANGLE graphics backend from the default value in edge://flags/#use-angle to "D3D11on12". This change fixed the incorrect image representation on Edge. How would one change this parameter on Jellyplayer? Is there a piece of code I could modify to change the ANGLE backend?
